File: app/src/main/java/com/paradoxo/amadeus/util/Toasts.java
package com.paradoxo.amadeus.util;

import android.content.Context;
import android.widget.Toast;

import static android.widget.Toast.LENGTH_LONG;
import static android.widget.Toast.LENGTH_SHORT;

public class Toasts {
    public static void meuToast(String texto, Context context) {
        Toast.makeText(context, texto, LENGTH_SHORT).show();
    }
    public static void meuToastLong(String texto, Context context) {
        Toast.makeText(context, texto, LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/paradoxo/amadeus/util/Toasts.kt
package com.paradoxo.amadeus.util

import android.content.Context
import android.widget.Toast

object Toasts {
    fun meuToast(texto: String, context: Context) =
        Toast.makeText(context, texto, Toast.LENGTH_SHORT).show()

    fun meuToastLong(texto: String, context: Context) =
        Toast.makeText(context, texto, Toast.LENGTH_LONG).show()
}
